src/mind.rs

```diff
@@ -134,6 +135,9 @@ pub fn α_β_negamax_search(world: WorldState,
         if experienced_α >= β {
             break;
         }
+        if (20000.0 - experienced_α.abs()).abs() < 1000.0 {
```

Comment:
(Unfortunately, since I had originally just imitated the pseudocode from *Wikipedia* rather than deeply understanding α-β search as a theorem, it took me a while to convince myself that this is correct. But, yes, it does look correct: α is the top score we're already assured of (in this "negamax" implementation, we are always the maximizing player), and if that's in the vicinity of 20000, that means an ultimate endangerment scenario has been found, and we don't need to search anymore. Well done.)